### PR TITLE
Revised the toggleActive function

### PR DESCRIPTION
**Pull request by viennguyen2026**
## Overview

Revised the toggleActive function to address Derek's comment on PR #62 

Closes #62

## Notes

I've tested this and it works on my machine

### CODE REVIEW

File: js/map.js [hunk 1]
```diff
@@ -285,36 +296,44 @@ map.on('load', () => {
   showLayer(selectedGeography, selectedCategory, selectedStatus)
   updateLegend(selectedGeography, selectedCategory, selectedStatus)
 
+  // calls the toggleActive function on the initial states
```
Comment (derekeder) [inline js/map.js:299]: we still need to handle the case when the map is loaded with different values set. each time a button is clicked it updates the URL with query parameters. we need to update the code here to account for selecting the right button

**Review by derekeder — CHANGES_REQUESTED** [review]
@viennguyen2026 this is looking good! works as expected when toggling the different buttons. one case to address though

**Comment by viennguyen2026** [comment]
@derekeder pushed based on the changes discussed above

**Review by derekeder — APPROVED** [review]
sorry this sat for so long @viennguyen2026 - looks good!

**Pull request merged**